src/components/timelapse/PlayButton.vue

```diff
@@ -24,6 +24,11 @@ export default {
     eventBus.$on("dates-selected", () => {
       this.isPlaying = false; // pause the button (the bar has already been paused in TimeLapseBar)
     });
+    eventBus.$on("remote-pause", () => {
+      if (this.isPlaying) {
```

Comment:
Makes me think that we should watch the `isPlaying` property and emit the `eventBus.$emit("toggle-timelapse", this.isPlaying);` in there. That will also allow us to get rid of duplicate code between lines 116-117 and 122-123 in `TimelapseBar.vue`. But yeah that can be a future refactor.